spec/spec.md

```diff
@@ -68,24 +72,44 @@ on GitHub (see above) and other mediums (e.g. DIF) where this work is being done
+| DID Controller                 | Entity that is authorized to make changes to the contents of a DID Document. [[spec:DID-CORE]].                                                                                                              |
 | DID Document                   | A set of data describing the DID subject, service and verification methods, that the DID subject or a DID delegate can use to authenticate itself and prove its association with the DID. [[spec:DID-CORE]]. |
+| DID Method                     | A definition of how a specific DID scheme implementeds the precise operations by which DIDs are created, resolved and deactivated and DID documents are written and updated. [[spec:DID-SPEC-REGISTRIES]]    |
 | Verifiable Credential          | A cryptographically secure mechanism for expressing credentials like driver's licenses on the web. It is defined by [[spec:VC-DATA-MODEL]].                                                                  |
 | Verifiable Presentation        | A way to combine and present credentials. It is defined by [[spec:VC-DATA-MODEL]].                                                                                                                           |
 
 ## Linked Verifiable Presentation Service Endpoint
 
-To enable discovery of linked verifiable presentations from the resolution of a DID, there must exist a DID Document
-mechanism for expressing locations where Verifiable Presentations MAY be located. To that end, the following section
-codifies the `LinkedVerifiablePresentations` Service Endpoint, which allows a DID controller to specify a list of
-presentations over which they assert control. Presentations asserted within the `LinkedVerifiablePresentations` endpoint
-descriptor can then be subsequently crawled by verifying parties to locate and verify any Verifiable Presentation
-resources that may exist.
+To enable the public discovery of verifiable presentations and verifiable credentials for a DID, there must exist a DID
+Document mechanism for expressing locations where verifiable presentations MAY be located. To that end, the following
+section codifies the `LinkedVerifiablePresentations` service endpoint. The endpoint allows a DID controller to publish
+and link verifiable presentations with verifiable credentials that SHALL be publicly associated with the DID.
+Presentations referenced within the `LinkedVerifiablePresentations` endpoint descriptor can then be crawled by verifying
+parties to locate and verify any presentation resources that may exist.
 
-### Example
+The DID Controller MAY include the parameters `challenge` and `domain` in the URL of the service endpoint and proof of
+the referenced Verifiable Presentation to mitigate replay attacks. If present, a [verifier][verifier] SHOULD include the
```

Comment:
> If present, a [verifier][verifier] SHOULD include

This should be a MUST imo

Reply:
I do wonder if replay attacks are of concern in the case of public presentations.. Will think more on this

Reply:
> > If present, a [verifier][verifier] SHOULD include
> 
> This should be a MUST imo

I tried to mimic the VC spec that doesn't tell the implementer what they must do when it comes to the verification. Instead, the spec focuses on enforcing the data model / data types. What do you think?

Reply:
> I do wonder if replay attacks are of concern in the case of public presentations.. Will think more on this

Maybe there's a better term for it. The issue I'm trying to mitigate is that a hosting provider could swap out the presentation for an old one without anyone noticing, i.e. "replay" an old presentation to the public audience.